schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE DATABASE comments;
+USE comments;
+
+CREATE TABLE comment (
+    id integer not null auto_increment,
+    text varchar(1000) not null,
+    createdAt timestamp not null,
+    songtime integer not null,
+    PRIMARY KEY (id),
+    FOREIGN KEY (artistId)
+        REFERENCES artists(id),
+    FOREIGN KEY (songId)
+        REFERENCES songs(id)  
+);
+
+CREATE TABLE reply (
+    id integer not null auto_increment,
+    text varchar(1000) not null,
+    createdAt timestamp not null,
+    PRIMARY KEY (id),
+    FOREIGN KEY (commentId)
+        REFERENCES comments(id),
+    FOREIGN KEY (artistId)
+        REFERENCES artists(id),
+    FOREIGN KEY (songId)
+        REFERENCES songs(id)    
+);
+
+CREATE TABLE songs (
+    id integer not null,
+    PRIMARY KEY (id),
+    plays integer,
+    likes integer,
+    reposts integer,
+);
+
+CREATE TABLE artists (
+    id integer not null,
+    PRIMARY KEY (id),
+    name varchar(20),
+    imageURL varchar(65535),
```

Comment:
Similar comment with the text - this can be changed to text/longblob/blob/etc. i'm not too familiar with the best practices but worth a look.